src/oc_erchef/apps/oc_chef_wm/test/chef_wm_status_tests.erl

```diff
@@ -34,11 +34,13 @@
 setup_env() ->
     application:set_env(oc_chef_wm, health_ping_timeout, ?PING_TIMEOUT),
     application:set_env(oc_chef_wm, health_ping_modules, ?CHECK_MODS),
+    application:set_env(oc_chef_wm, include_version_in_status, true),
```

Comment:
Could you also include a test in which this is configured `false`, so that we can verify correct behavior in that case too? 

Separately - is there an `oc-chef-pedant` test for the `/status` endpoint which needs modification or additional coverage too?

Reply:
Sure. I will check and add the test cases. Thank you!